service/service.go

```diff
@@ -0,0 +1,63 @@
+package service
+
+import (
+	pb "github.com/faraonc/hwsc-api-blocks/int/hwsc-user-svc/proto"
+	"github.com/faraonc/hwsc-user-svc/logtag"
+	"golang.org/x/net/context"
+	"log"
+)
+
+// This Service struct type, implements the generated (pb file) UserServiceServer interface
+type Service struct{}
+
+func (s *Service) GetStatus (ctx context.Context, req *pb.UserRequest) (*pb.UserResponse, error) {
+	//TODO
+	logRequestService("GetStatus")
+	return nil, nil
+}
+
+func (s *Service) CreateUser (ctx context.Context, req *pb.UserRequest) (*pb.UserResponse, error) {
+	// TODO
+	logRequestService("CreateUser")
+	return nil, nil
+}
+
+func (s *Service) DeleteUser (ctx context.Context, req *pb.UserRequest) (*pb.UserResponse, error) {
+	//TODO
+	logRequestService("DeleteUser")
+	return nil, nil
+}
+
+func (s *Service) UpdateUser (ctx context.Context, req *pb.UserRequest) (*pb.UserResponse, error) {
+	//TODO
+	logRequestService("UpdateUser")
+	return nil, nil
+}
+
+func (s *Service) AuthenticateUser (ctx context.Context, req *pb.UserRequest) (*pb.UserResponse, error) {
+	//TODO
+	logRequestService("AuthenticateUser")
+	return nil, nil
+}
+
+func (s *Service) ListUsers (ctx context.Context, req *pb.UserRequest) (*pb.UserResponse, error) {
+	//TODO
+	logRequestService("ListUsers")
+	return nil, nil
+}
+
+func (s *Service) GetUser (ctx context.Context, req *pb.UserRequest) (*pb.UserResponse, error) {
+	//TODO
+	logRequestService("GetUser")
+	return nil, nil
+}
+
+func (s *Service) ShareDocument (ctx context.Context, req *pb.UserRequest) (*pb.UserResponse, error) {
+	//TODO
+	logRequestService("ShareDocument")
+	return nil, nil
+}
+
+func logRequestService (svc string) {
```

Comment:
Please see comment above - https://github.com/faraonc/hwsc-logging

Reply:
Make sure you implement something similar to ``fmt.Printf()`` for formatting strings. In that way we can simplify our func parameters.
https://gobyexample.com/string-formatting

There is a concept called variadic functions in GoLang.
https://gobyexample.com/variadic-functions

This is worth looking into as well for building Strings. 
https://golang.org/pkg/fmt/#Sprintf